Header header 
int32[] storage
int32[] data_stimulus 
int32[] buffered_regular_objects
int32[] buffered_priority_objects
   